br-webui/views/network.liquid

```diff
@@ -296,5 +370,19 @@ socket.on('current ip', function(data) {
 		</div>
 	</div>
 </div>
+<script>
 
+function onToggleAdvancedSwitch() {
+	var div = document.getElementById("eth-conf");
```

Comment:
Instead of going by id, and having to put down every new element we add right here, we should add a unique identifier to the DOM elements that should be affected by this.
Then, use [getElementsByClassName](https://www.w3schools.com/jsref/met_document_getelementsbyclassname.asp) or jquery to select the elements to hide/unhide, and iterate through them